cmd/uplink/cmd/setup.go

```diff
@@ -247,3 +272,39 @@ func ApplyDefaultHostAndPortToAddr(address, defaultAddress string) (string, erro
 	// address is host:
 	return net.JoinHostPort(addressParts[0], defaultPort), nil
 }
+
+// saveEncryptionKey generates a Storj key from the inputKey and save it into a
+// new file created in filepath.
+func saveEncryptionKey(inputKey []byte, filepath string) error {
```

Comment:
This still doesn't do what I requested. This method should take the human-readable encryption key and just save it to the file as is, in plain text. The input shouldn't be changed in any way, like passing it to `storj.NewKey` or anything else.

Reply:
oh, I misunderstood your request